com.microsoft.mrtk.graphicstools.unity/Editor/ShaderGUIs/StandardShaderGUI.cs

```diff
@@ -894,10 +894,31 @@ protected void FluentOptions(MaterialEditor materialEditor, Material material)
                     }
                 }
             }
+            
+            if (!PropertyEnabled(roundCorners) && PropertyEnabled(borderLight))
+            {
+                materialEditor.ShaderProperty(edgeSmoothingMode, Styles.edgeSmoothingMode, 2);
+
+                switch ((EdgeSmoothingMode)edgeSmoothingMode.floatValue)
+                {
+                    default:
+                    case EdgeSmoothingMode.Manual:
+                    {
+                        material.DisableKeyword(Styles.edgeSmoothingModeAutomaticName);
+                        materialEditor.ShaderProperty(edgeSmoothingValue, Styles.edgeSmoothingValue, 3);
+                    }
+                        break;
+                    case EdgeSmoothingMode.Automatic:
+                    {
+                        material.EnableKeyword(Styles.edgeSmoothingModeAutomaticName);
+                    }
+                        break;
+                }
+            }
 
             if (PropertyEnabled(hoverLight) || PropertyEnabled(proximityLight) || PropertyEnabled(borderLight))
             {
-                materialEditor.ShaderProperty(fluentLightIntensity, Styles.fluentLightIntensity);
+                materialEditor.ShaderProperty(fluentLightIntensity, Styles.fluentLightIntensity, 2);

```

Comment:
One design suggestion, I wonder if it makes sense to move this property to the top?

![image](https://user-images.githubusercontent.com/13305729/210439987-a611f5dc-887d-40ca-b95a-25d54c292c8c.png)

That way it looks like it belongs to Hover Light, Proximity Light, and Border Light, rather than just Border Light.